src/cmd/build.js

```diff
@@ -103,15 +111,49 @@ async function defaultPackageCreator({
   let packageName = safeFileName(
     `${extensionName}-${manifestData.version}.zip`);
   let extensionPath = path.join(artifactsDir, packageName);
-  let stream = createWriteStream(extensionPath);
 
-  stream.write(buffer, () => stream.end());
+  let destinationFileExists = await fs.stat(extensionPath)
+    .then((stat)=>{
```

Comment:
to keep it consistent, we put a space between arrow function operators. I fixed the linting rules in https://github.com/mozilla/web-ext/pull/626 to catch this. If you merge in master it will show you how to fix everything.